static/js/stream_create.js

```diff
@@ -116,48 +116,71 @@ class StreamNameError {
 const stream_name_error = new StreamNameError();
 
 // Stores the previous state of the stream creation checkbox.
-let stream_announce_previous_value =
+let announce_stream_previous_value =
```

Comment:
If we actually need this change, this should be done in a separate commit.

Reply:
Made this change since we were using `announce_stream` everywhere else. Reverting it since not mandatory.